bpfctl/src/main.rs

```diff
@@ -43,7 +43,28 @@ enum Commands {
     /// Unload a BPF program using the UUID.
     Unload(UnloadArgs),
     /// List all BPF programs loaded via bpfd.
-    List,
+    List(ListArgs),
+    /// Get a program's metadata information specified by kernel id.
+    Get { kernel_id: u32 },
```

Comment:
nit: Since the only parameter is defined here and not in a struct below, there is no helptext for `kernel_id`:

```
$ bpfctl get --help
Get a program's metadata information specified by kernel id

Usage: bpfctl get <KERNEL_ID>

Arguments:
  <KERNEL_ID>  

Options:
  -h, --help  Print help
```

Reply:
fixed to 

```
[astoycos@localhost bpfd]$ ./target/debug/bpfctl get --help
Get a program's metadata information specified by kernel id

Usage: bpfctl get <KERNEL_ID>

Arguments:
  <KERNEL_ID>  A BPF program's kernel id

Options:
  -h, --help  Print help
```

Is that alright?

Reply:
Yes